### ISSUE TITLE
flatpak-builder --install leads to "error: Flatpak system operation ConfigureRemote not allowed for user"

### ISSUE THREAD

**Issue by mwleeds** [flatpak/flatpak-builder#290]
## Linux distribution and version
Endless OS 3.6.0 and Fedora Silverblue 31
## Flatpak-builder version
1.0.6 and 1.0.7
## Flatpak version
1.4.0
## Description of the problem
Using the `--install` option with flatpak-builder leads to an error (unless `--user` is used) instead of actually installing the flatpak.
## Steps to reproduce
1. `git clone https://github.com/flathub/org.kde.kwordquiz`
2. `cd org.kde.kwordquiz`
3. `flatpak-builder --force-clean --install --install-deps-from=flathub _build org.kde.kwordquiz.json`
At the end of the output is:
```
error: Flatpak system operation ConfigureRemote not allowed for user
Install failed: Child process exited with code 1
```

**Comment by mwleeds** [comment]
Looks like the `ConfigureRemote` operation being attempted is to add a `kwordquiz-origin` remote.

**Comment by mwleeds** [comment]
We're passing `-y --noninteractive` to the `flatpak install` command in `builder-main.c`. If I remove those I get an authentication prompt and then the install fails when there's no input to this prompt:
`Proceed with these changes to the Default system installation? [Y/n]: n`

So I guess there are three options: we either
1. document that `--install` doesn't work with system installations that require authentication, or
2. change the rules in the system helper so that you don't need to authenticate to add an origin remote (which could pose a security risk), or
3. if `flatpak-builder` can be interactive, allow the authentication prompt through while still passing `-y` to `flatpak install` (which would require changes to how `-y` is interpreted).

**Comment by refi64** [comment]
Random thought, option 1 could be a bit inconsistent since of course it works if you're in the wheel group.

**Comment by mwleeds** [comment]
> Random thought, option 1 could be a bit inconsistent since of course it works if you're in the wheel group.

No, by default even sudoers can't add a remote without authenticating. They can only install from existing remotes.

**Comment by refi64** [comment]
Hmm that's odd, I'm quite sure I've run flatpak-builder --force several times without needing to auth, unless it's changed since then or it's a Fedora-ism?

**Comment by mwleeds** [comment]
Did you mean `--install`?

**Comment by refi64** [comment]
Sorry, yeah.

**Comment by mwleeds** [comment]
There have been relatively recent changes to how Flatpak handles non-interactive installs, so it might be that you're not on 1.4.x

**Comment by Erick555** [comment]
@mwleeds I just hit the same issue. Did you found any workaround for installing flatpak system-wide?

**Comment by Erick555** [comment]
I ended up with creating bundle and installing it with` flatpak install`.

**Comment by gnumoksha** [comment]
I've added `--user` argument and it worked. 

I'm running:
```console
$ flatpak --version
Flatpak 1.6.2
$ flatpak-builder --version
flatpak-builder 1.0.9
```

**Comment by Erick555** [comment]
@gnumoksha yes, `--user` works but this issue is about system installations.

**Comment by Newbytee** [comment]
> I ended up with creating bundle and installing it with` flatpak install`.

How did you do that?

**Comment by Erick555** [comment]
You need to use `flatpak build-bundle` command. See [manpage](https://man.archlinux.org/man/extra/flatpak/flatpak-build-bundle.1.en) for available options.
